script/run-unit-test.sh

```diff
@@ -9,12 +9,12 @@ if [ ! -f $MOCHA ]; then
 fi
 
 # No test specified; run all of them
-if [ -z "$1" ]; then
-  BABEL_ENV=test $MOCHA test/helper.js --recursive '$SCRIPT_DIR/../test/**/*.unit.spec.js?(x)' 
+if [[ -z "$1" ]] || [[ $1 == -* ]]; then
```

Comment:
@cvalarida I made some changes here so that we can pass the debugging flags to the mocha command and debug tests in dev tools again.

Reply:
Ohh, yeah, that's much nicer!

I wrote this before I knew the magic bash incantation `"$@"`. Looks great! :+1: